src/View/Components/Choices.php

```diff
@@ -163,10 +154,15 @@ public function render(): View|Closure|string
                                         : this.selection.push(id)
                                 }
 
-                                this.onChange(this.selection)
-
                                 this.$refs.searchInput.value = ''
                                 this.$refs.searchInput.focus()
+                                this.$refs.searchInput.dispatchEvent(new CustomEvent('change', {
+                                    bubbles: true, 
+                                    detail: {
+                                        target: this.$refs.searchInput,
+                                        value: this.selection
+                                    }
+                                }))
```

Comment:
The simple one does not work as expected?

this.$dispatch

Reply:
It does not work quite as expected. You cannot listen for the event on the input itself. You have to listen on the window with something like x-on:change.window. And the reason I wanted to add this feature was for the change event to feel like a normal browser event, like you would listen for with any other input. With this approach, you can do something like @change="console.log(e)" right on the choices element, and you can capture the event and do stuff with it. It is a custom event so it isn't quite the same as a normal browser event, but it works

I think dispatching a livewire event would be great, but it is also an extra feature that I wasn't trying to add with this pull request. I just wanted to add a feature that would mimic a browser event.